cypress/integration/required-metadata.spec.js

```diff
@@ -73,13 +74,14 @@ describe('Required Metadata Page', () => {
       .contains('Save and Continue')
       .click()
       .then(() => {
-        cy.visit('/dataset/a-title-for-tag-tab-test1');
+        const name = 'a-title-for-tag-tab-test1';
+        cy.visit(`/dataset/${name}`);
         cy.contains(tagName);
+        cy.request('POST', '/api/3/action/dataset_purge', { id: name });
```

Comment:
trying to get this, I see it in other test cases too, why do we need this?

Reply:
@kmanaseryan we need to purge datasets that have static names after tests completed so that re-running tests wouldn't cause any failures (eg, with "already exists" error)